### DISCUSSION TITLE
Colour scheme

### DISCUSSION THREAD

**Discussion by thomasMatson** [thithuongmai/T24S2T1EXTG05#3] [General]
Should we have a unified colour scheme for all calculators, or a unique colour for each to differentiate them

If unified, what colour should we use?

**Comment by thithuongmai** [comment]
Yes, I think so because this can provide a consistent look across all calculators, making the app easier to navigate and giving a professional feel.
Base on the Main Menu mockup screenshot in the requirement, I would suggest below colours: 
- Background: LightYellow
- Title : Blue
- Text : black
- Button background: SandyBrown
